### ISSUE TITLE
Unselectable Text in Preview Pane

### ISSUE THREAD

**Issue by Gazook89** [naturalcrit/homebrewery#3709]
The Vault page css is applying styles globally that shouldn't be. 

https://github.com/naturalcrit/homebrewery/blob/780c92cb9b53ba4cd40d14e6f0fd44e292eaeced/client/homebrew/pages/vaultPage/vaultPage.less#L1C1-L20C2

These lines are applied to every page, not just the Vault page, so things like `user-select: none` are preventing text selection in the Preview Pane (the `body` tag of the iframe).  It is also explains the odd color choice in the Notifications that @5e-Cleric pointed out in Gitter earlier:

![image](https://github.com/user-attachments/assets/74b75d21-4cc7-4ace-bc8e-04d459bc1e20)

Basically I don't think any of this CSS should be declared just under `body` so broadly-- it needs to be moved under `.vaultPage` and applied much more narrowly.
